### ISSUE TITLE
ESP8266 build broken

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#842]
current master fails building esp8266

```
GEN build/genhdr/qstr.i.last
QSTR updated
GEN build/genhdr/qstrdefs.generated.h
../../py/moduerrno.c: In function 'mp_errno_to_str':
../../py/moduerrno.c:106:18: error: 'EPERM' undeclared (first use in this function)
             case EPERM:  return MP_QSTR_Permission_space_denied;
                  ^
../../py/moduerrno.c:106:18: note: each undeclared identifier is reported only once for each function it appears in
../../py/moduerrno.c:107:18: error: 'ENOENT' undeclared (first use in this function)
             case ENOENT: return MP_QSTR_No_space_such_space_file_slash_directory;
                  ^
../../py/moduerrno.c:108:18: error: 'EIO' undeclared (first use in this function)
             case EIO:    return MP_QSTR_Input_slash_output_space_error;
                  ^
../../py/moduerrno.c:109:18: error: 'EACCES' undeclared (first use in this function)
             case EACCES: return MP_QSTR_Permission_space_denied;
                  ^
../../py/moduerrno.c:110:18: error: 'EEXIST' undeclared (first use in this function)
             case EEXIST: return MP_QSTR_File_space_exists;
                  ^
../../py/moduerrno.c:111:18: error: 'ENODEV' undeclared (first use in this function)
             case ENODEV: return MP_QSTR_Unsupported_space_operation;
                  ^
../../py/moduerrno.c:112:18: error: 'EINVAL' undeclared (first use in this function)
             case EINVAL: return MP_QSTR_Invalid_space_argument;
                  ^
../../py/mkrules.mk:53: recipe for target 'build/py/moduerrno.o' failed
make: *** [build/py/moduerrno.o] Error 1

```

**Comment by jerryneedell** [comment]
OK simple fix:
```
jerryneedell@Ubuntu-Macmini:~/projects/adafruit_github/circuitpython_jerryn/ports/esp8266$ git diff
diff --git a/ports/esp8266/mpconfigport.h b/ports/esp8266/mpconfigport.h
index 616e987..907a4cb 100644
--- a/ports/esp8266/mpconfigport.h
+++ b/ports/esp8266/mpconfigport.h
@@ -28,7 +28,7 @@
 #define MICROPY_ENABLE_SOURCE_LINE  (1)
 #define MICROPY_MODULE_WEAK_LINKS   (1)
 #define MICROPY_CAN_OVERRIDE_BUILTINS (1)
-#define MICROPY_USE_INTERNAL_ERRNO  (1)
+#define MICROPY_USE_INTERNAL_ERRNO  (0)
 #define MICROPY_ENABLE_SCHEDULER    (1)
 #define MICROPY_PY_ALL_SPECIAL_METHODS (1)
 #define MICROPY_PY_BUILTINS_COMPLEX (0)
```
I'll put in a PR.

**Comment by jerryneedell** [comment]
fixed by #843
